### DISCUSSION TITLE
About the function of saving after 3D cropping

### DISCUSSION THREAD

**Discussion by 865979216** [Kitware/VolView#419] [Q&A] [answered]
How to save the remaining 3D to my  computer after using the cropping function to trim 3D. 
For example, the current 3D, there is a hand, and after I crop it, there is only one finger left. How to save the dicom or  .vti of this finger to my computer

**Comment by floryst** [comment, marked as answer]
Cropping is currently a visual-only feature and not a data editing feature. Cropped export would be an interesting feature to add.

A workaround is to save the application session, read the cropping state out of the state file, then apply an ITK cropping filter.
